pkgs/by-name/wa/warp-terminal/package.nix

```diff
@@ -24,13 +88,18 @@ stdenvNoCC.mkDerivation (finalAttrs: {
 
     runHook postInstall
   '';
+});
 
-  meta = with lib; {
-    description = "Rust-based terminal";
-    homepage = "https://www.warp.dev";
-    license = licenses.unfree;
-    sourceProvenance = with lib.sourceTypes; [ binaryNativeCode ];
-    maintainers = with maintainers; [ emilytrau Enzime ];
-    platforms = platforms.darwin;
-  };
-})
+meta = with lib; {
+  description = "Rust-based terminal";
+  homepage = "https://www.warp.dev";
+  license = licenses.unfree;
+  sourceProvenance = with lib.sourceTypes; [ binaryNativeCode ];
+  maintainers = with maintainers; [ emilytrau Enzime ];
+  platforms = [ darwin "x86_64-linux" ];
```

Comment:
```
platforms = platforms.darwin ++ [ "x86_64-linux" ];
```
 I guess

Reply:
I 100% knew that when I was writing this and was just testing the reviewers 100000%%%%